src/App.jsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
-import { Switch, Route } from 'react-router-dom';
+import { Switch, Route, Redirect } from 'react-router-dom';
 import './App.scss';
-import Header from './pages/header/components/Header/Header';
 import Landing from './pages/home/components/Landing/Landing';
+import Routes from './shared/constants/paths/paths';
```

Comment:
Let's write imported constants in UPPERCASE, so they won't look like imported React components